docs/cluster-management/config.md

```diff
@@ -51,6 +51,7 @@ spot: false
 The docker images used by Cortex are listed below. They can be overridden to use custom images by specifying them in your cluster configuration file.
 
 You can follow these [instructions](../deployments/system-packages.md) to build and push custom Docker images to a registry and configure Cortex to use them.
+These images can also be configured on a per-API basis using `image` keys as documented in these [instructions](../deployments/api-configuration.md).
```

Comment:
As I mentioned below, I think may be best to actually drop the "user-facing" images from cluster configuration. I'd be happy to discuss, or if you agree, do you mind updating these docs accordingly? (I can also take a stab if you'd like)

Reply:
Yep, I'll update these as soon as the whole backend is taken care of.

Reply:
Yep. You're right.